azure-pipelines/signing.yml

```diff
@@ -17,13 +17,21 @@ parameters:
   displayName: vcpkg Base Version (default is today's date in ISO 8601)
   type: string
   default:
-
+- name: PublishLocation

```

Comment:
Could this be a different name? Something like `PublishedTo` or smth. `PublishLocation` makes me think of a URI to publish to or something.